github/projects.go

```diff
@@ -466,3 +466,127 @@ func (s *ProjectsService) MoveProjectCard(ctx context.Context, cardID int64, opt
 
 	return s.client.Do(ctx, req, nil)
 }
+
+type ProjectCollaboratorOptions struct {
+	// Permission specifies the permission to grant to the collaborator.
+	// Possible values are:
+	//     read - can read, but not write to or administer this project.
+	//     write - can read and write, but not administer this project.
+	//     admin - can read, write and administer this project.
+	//
+	// Default value "write"
+	// Note that, if you choose not to pass any parameters,
```

Comment:
After you address the comment for line 480 below, this note will no longer be true, so please delete lines 478 and 479.